from rest_framework.response import Response


def custom_error_handler(ext: Exception, content) -> Response:
    pass
